README.md

```
@@ -1,97 +1,112 @@
+- **회원가입**:
+  - 회원가입 시 DB에 유저정보가 등록됩니다.
+
+- **로그인**:
+  - 사용자 인증 정보를 통해 로그인합니다.
+
+<br/>
+<br/>
+
+# 5. 기술 스택 + 선정 이유
+## 5.1 Language
+|  |  |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)